### PR TITLE
Adds reminder email for mentors to sign in

### PR DESCRIPTION
**Pull request by xswoodsx**
## Ticket and context

Ticket: [801](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?selectedIssue=CPDEL-801)

We want to send a reminder email to all private beta mentors. It's going to be ran manually in the terminal with a list of emails we have already. 

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### Review Checks
- [ ] All commit messages are meaningful and true
- [ ] Added enough automated tests
- [ ] All forms have an `id` attribute on them

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Run it manually in the console. 

Screenshot 2021-09-24 at 14 19 00

### CODE REVIEW

File: app/models/reminder_email_mentor.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ReminderEmailMentor < TrackedEmail
+  def mail_to_send
+    UserMailer.mentor_sign_in_reminder_email(user)
+  end
+
+  def send!(force_send: false)
```
Comment (kicferk1) [inline app/models/reminder_email_mentor.rb:8]: If only a dev ever uses this, I am not sure we need the `force_send`. But if we do need it, unit tests please :)

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -64,4 +65,19 @@ def sign_in_email(user:, url:, token_expiry:)
       },
     )
   end
+
+  def mentor_sign_in_reminder_email(user)
+    sign_in_url = Rails.application.routes.url_helpers.new_user_session_url(host: Rails.application.config.domain,
+                                                                            **UtmService.email(:new_mentor))
```
Comment (kicferk1) [inline app/mailers/user_mailer.rb:71]: > **UtmService.email(:new_mentor))

Is this true? I think not. Let's make a new campaign for it, like `:mentor_sign_in_reminder`?

Comment (xswoodsx) [reply app/mailers/user_mailer.rb:71]: good spot! completely missed this.

File: app/models/reminder_email_mentor.rb (hunk 1 above)
Comment (xswoodsx) [reply app/models/reminder_email_mentor.rb:8]: I left this is thinking whether all the mentors be registered? I don't think they will. We can remove the `registered_participant` check, again left this in for potential future email sends. Don't mind either way but will add force_send tests

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**